app/routes/api.circulars.ts

```diff
@@ -100,7 +100,7 @@ async function getUserAttributes(Username: string) {
   }
 }
 
-async function getUserGroups(Username: string) {
+export async function getUserGroups(Username: string) {
```

Comment:
Since we are resuing this function in multiple routes now, move this to a library module like cognito.server.ts.

Reply:
I actually don't think this was intended to be modified, but I will refactor it to use some of the new functions since it is basically doing the same thing anyway